Topics/Development_Process/Vue_Test_Utils.md

```diff
@@ -0,0 +1,55 @@
+# Intro to Vue Test Utils (VTU)
+Vue Test Utils (commonly abbreviated as VTU) are a set of functions that make it easier to test Vue.js components.  It is commonly used
+with a test runner for automated testing in software engineering projects.
+
```

Comment:
It would be nice to elaborate a bit more in the intro section to give the reader a clearer picture of what VTU is and what exactly does it accomplish. You can consider adding some examples of useful functionalities of VTU. Also would be a good idea to include some images to accompany the text

Reply:
Good idea, I haven't thought about doing that.